tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -329,6 +329,80 @@ def test_derive_code_mappings_starts_with_app_dot_dot_slash(self):
             assert code_mapping.repository.name == repo_name
 
 
+class TestPhpDeriveCodeMappings(BaseDeriveCodeMappings):
+    def setUp(self):
+        super().setUp()
+        self.platform = "php"
+        self.event_data = self.generate_data(
+            [
+                {"in_app": True, "filename": "/sentry/capybara.php"},
+                {"in_app": True, "filename": "/sentry/potato/kangaroo.php"},
+                {
+                    "in_app": False,
+                    "filename": "/sentry/potato/vendor/sentry/sentry/src/functions.php",
+                },
+            ],
+            self.platform,
+        )
+
+    def test_find_stacktrace_paths_single_project(self):
```

Comment:
You don't need to add these two tests. This is already being tested in other tests in this module.